cc/mcts_player.h

```diff
@@ -194,7 +194,22 @@ class MctsPlayer {
     int last_move = 0;
   };
 
-  void GetModelInput(const MctsNode* node, Model::Input* input) const;
+  symmetry::Symmetry GetCanonicalSymmetry(const MctsNode* node) const {
```

Comment:
commenting the difference between canonical/inference is useful.

Reply:
done.